src/groups/rotation_action.jl

```diff
@@ -26,11 +26,11 @@ function Base.show(io::IO, A::RotationAction)
     return print(io, "RotationAction($(A.manifold), $(A.SOn), $(direction(A)))")
 end
 
-const RotationActionOnVector{N,F,TAD} = RotationAction{
-    <:Union{Euclidean{Tuple{N},F},TranslationGroup{Tuple{N},F}},
-    SpecialOrthogonal{N},
+const RotationActionOnVector{TAD,𝔽,TE,TSO} = RotationAction{
```

Comment:
You turned these around since then the dispatch below is easier?

Reply:
Yes, exactly.